src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2919,10 +2977,14 @@ void InitSolarHeatGains(EnergyPlusData &state)
                                     state.dataEnvrn->DifSolarRad * state.dataSolarShading->SurfAnisoSkyMult(OutShelfSurf)) *
                                    state.dataDaylightingDevicesData->Shelf(ShelfNum).OutReflectSol;
 
+            Real64 GndReflSolarRad = state.dataEnvrn->GndSolarRad;
+            if (state.dataSurface->UseSurfPropertyGndSurfRefl(SurfNum)) {
+                GndReflSolarRad = state.dataSurface->GndReflSolarRad(SurfNum);
+            }
```

Comment:
Resets the ground surfaces reflected solar radiation if `SurfaceProperty:GroundSurfaces` object is specified for a given exterior surface.